SKILL.tsv

```diff
@@ -3791,72 +3791,72 @@ SKILL_20150414_003790	Temporarily hide yourself away from enemy's vision. You be
 SKILL_20150414_003827	$Creates a magic circle that decreases the defense of enemies over time. 
 SKILL_20150414_003828	{memo X}$Creates a magic circle that increases the skill level of allies once.
 SKILL_20150414_003829	{memo X}$Summon a holy torch to improve the HP recovery rate of nearby allies.
-SKILL_20150414_003830	$Creates a magic circle that improves Critical of allies or worsen Critical of enemies.
-SKILL_20150414_003831	$Increases the number of beneficial effects that you and your party members can receive.
-SKILL_20150414_003832	${#993399}{ol}[Magic] - [Lightning]{/}{/}{nl}Creates a magic circle that releases bolts of lightning using the power of the god of Thunder that strike down an enemy.
-SKILL_20150414_003833	$Engraves a stigma on an enemy. Increases your and your party member's STR and INT when the engraved enemy has been defeated.
-SKILL_20150414_003834	$Creates a magic circle that helps you to maintain the beneficial effects of you and your party members.
+SKILL_20150414_003830	Create a magic circle that improves the critical rate of allies or decreases the critical rate of enemies.
+SKILL_20150414_003831	Increases the number of beneficial effects that you and your party members can receive.
+SKILL_20150414_003832	{#993399}{ol}[Magic] - [Lightning]{/}{/}{nl}Using the God of Thunder's power, create a magic circle that calls down bolts of lightning.
+SKILL_20150414_003833	Engraves the divine stigma on an enemy. You and your party member's strength and intelligence are increased when the engraved enemy has been defeated.
+SKILL_20150414_003834	Create a magic circle that temporarily maintains the beneficial effects of you and your party members.
 SKILL_20150414_003835	{memo X}${#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Sprinkle holy water to increase the defense of you and your allies as well as inflict damage to enemies.
 SKILL_20150414_003836	{memo X}Create magic circle where ally's DEX increase and enemy's ecasion and Physical defense decrease.
 SKILL_20150414_003837	{memo X}Give blessings that grant additional damage on attacks of party members.
-SKILL_20150414_003838	$Resurrects a target in the state of Incapable of Combat.
-SKILL_20150414_003839	$Temporarily grants the Holy property to you and your party member's attacks.
-SKILL_20150414_003840	{#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Create exorcism circle that attacks enemies stepping it then give continuous damage.
-SKILL_20150414_003841	Curse enemy to decrease its magic defense and prevent recovery.
-SKILL_20150414_003842	{#993399}{ol}[Magic] - [Dark]{/}{/}{nl}Control enemy under curse and inflict damage. Bonus damage applies on 3rd attack.
-SKILL_20150414_003843	Use monkey skull to lure zombies to desired area.
+SKILL_20150414_003838	Resurrects a target that has been defeated in combat.
+SKILL_20150414_003839	Temporarily grants a holy property to you and your party member's attacks.
```

Comment:
There is nothing wrong with this line.

It mentions the Holy property, which is one of the properties in TOS.